### PR TITLE
Create issue and pull request templates

### LINKED ISSUE #49: Create issue and pull request templates

### PR DESCRIPTION
**Pull request by ChrispyPeaches**
Fixes #49

## Problem
We need consistent formatting for pull requests and issues

## Solution
Create templates

### Pull Request
image

### Issue
image

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Problem
+<!--Add a description and the userstory prompt-->
+
+## Solution
```
Comment (JohnParks032) [inline .github/issue_template.md:4]: Would it make more sense if we didn't have a solution on the issue template? My reasoning is that there's a chance that implementation ends up differently than initially thought.

**Review by JohnParks032 — CHANGES_REQUESTED** [review]
Made small comment on issue template

File: .github/issue_template.md (hunk 1 above)
Comment (JohnParks032) [reply .github/issue_template.md:4]: I realized that this could also be a feature and show us how our idea could've changed/developed.

**Review by JohnParks032 — APPROVED** [review]
LGTM

**Review by wesleypayton — APPROVED** [review]
LGTM 💯

**Pull request merged**